testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/jpa2lc/DummyEntity.java

```diff
@@ -27,8 +27,12 @@
 import javax.persistence.Entity;
 import javax.persistence.Id;
 
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
 @Entity
 @Cacheable
+@Cache(usage = CacheConcurrencyStrategy.TRANSACTIONAL)
```

Comment:
Since this test doesn't modify data, I think READ_ONLY could be used instead of TRANSACTIONAL (some doc on different modes are in https://docs.jboss.org/hibernate/orm/5.1/userguide/html_single/chapters/caching/Caching.html).  

@sebersole did we ever try to get a javax.persistence.CacheConcurrencyStrategy added to the JPA spec?  I don't recall that coming up in the past.

Reply:
I am just going to merge this and see if it helps the test, if it keeps failing I will ignore it.